openrota/openrota-database-template.yaml

```diff
@@ -0,0 +1,213 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+labels:
+  app: openrota
+  paas.redhat.com/appcode: ${APP_CODE}
+message: |-
+  The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.
+
+         Username: ${POSTGRESQL_USER}
+         Password: ${POSTGRESQL_PASSWORD}
+    Database Name: ${POSTGRESQL_DATABASE}
+   Connection URL: postgresql://${DATABASE_SERVICE_NAME}:5432/
+
+  For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/.
+metadata:
+  name: openrota-database-template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: ${DATABASE_SERVICE_NAME}
+    labels:
+      tier: database
+  stringData:
+    database-name: ${POSTGRESQL_DATABASE}
+    database-password: ${POSTGRESQL_PASSWORD}
+    database-user: ${POSTGRESQL_USERNAME}
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${DATABASE_SERVICE_NAME}
+    labels:
+      tier: database
+  spec:
+    ports:
+      - name: postgresql
+        nodePort: 0
+        port: 5432
+        protocol: TCP
+        targetPort: 5432
+    selector:
+      name: ${DATABASE_SERVICE_NAME}
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      kubernetes.io/reclaimPolicy: Delete
```

Comment:
reclaimPolicy should not be delete. Please refer https://source.redhat.com/groups/public/itpaas/managedplatformplushub/mppluswiki/storage_offerings

Reply:
yeah...lets go ahead with "Retain" considering reclaimPolicy is required -> "An annotation reclaimPolicy of either “Delete” or “Retain” is required and failure to include will lead to the storage webhook to deny your request."